app/controllers/concerns/filter_by_owner.rb

```diff
@@ -8,7 +8,7 @@ module FilterByOwner
   def add_owner_ids_to_filter_param!
     owner_filter = params.delete(:owner).try(:split, ',')
     unless owner_filter.blank?
-      groups = UserGroup.where(name: owner_filter)
+      groups = UserGroup.where('lower(name) = ?', owner_filter.map(&:downcase))
```

Comment:
Ah, right the specs pass because there's only a single filter. If you'd add a second owner name, you'd see the specs fail with a SQL syntax error:

```
[9] pry(main)> puts UserGroup.where("lower(name) = ?", ['a', 'b'])
  UserGroup Load (1.6ms)  SELECT "user_groups".* FROM "user_groups" WHERE (lower(name) = 'a','b')
PG::DatatypeMismatch: ERROR:  argument of WHERE must be type boolean, not type record
LINE 1: SELECT "user_groups".* FROM "user_groups" WHERE (lower(name)...
                                                        ^
: SELECT "user_groups".* FROM "user_groups" WHERE (lower(name) = 'a','b')
ActiveRecord::StatementInvalid: PG::DatatypeMismatch: ERROR:  argument of WHERE must be type boolean, not type record
LINE 1: SELECT "user_groups".* FROM "user_groups" WHERE (lower(name)...
                                                        ^
: SELECT "user_groups".* FROM "user_groups" WHERE (lower(name) = 'a','b')
from /Users/marten/.rvm/gems/ruby-2.2.1/gems/activerecord-4.2.2/lib/active_record/connection_adapters/postgresql_adapter.rb:592:in `async_exec'
```